### PR TITLE
Assignment6

### PR DESCRIPTION
**Pull request by mcstudor**

### CODE REVIEW

**Comment by JerettLatimer** [comment]
Will Review!

File: SecretSanta/src/SecretSanta.Api/appsettings.json [hunk 1]
```diff
@@ -6,5 +6,8 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "DefaultConnection": "Data Source=SantaProd.db"
```
Comment (JerettLatimer) [inline SecretSanta/src/SecretSanta.Api/appsettings.json]: Database string moved into appsettings. Nice!

**Comment by JerettLatimer** [comment]
Looks good, just missing the delete pages.

File: SecretSanta/src/SecretSanta.Web/SecretSanta.Web.csproj [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.1" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="TagHelperPack" Version="0.5.0-pre-20191217.4" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="scripts\" />
+  </ItemGroup>
+    <ItemGroup>
+      <TypeScriptCompile Include="scripts\main.ts" />
```
Comment (breaman) [inline SecretSanta/src/SecretSanta.Web/SecretSanta.Web.csproj]: This should not be here since we are using webpack to compile the typescript

**Pull request closed without merging**